component/distributed_cache/distributed_cache.go

```diff
@@ -1031,7 +1031,8 @@ func (dc *DistributedCache) OpenFile(options internal.OpenFileOptions) (*handlem
 	}
 
 	// todo: We should only support write if the file is only in Azure.
-	if options.Flags&os.O_WRONLY != 0 || options.Flags&os.O_RDWR != 0 {
+	if (options.Flags&os.O_WRONLY != 0 || options.Flags&os.O_RDWR != 0) &&
+		isDebugPath && rawPath != "logs" { // We support write only for fs=debug/logs
```

Comment:
this opens up write for non-debug files, correct check should be

    if (options.Flags&os.O_WRONLY != 0 || options.Flags&os.O_RDWR != 0) &&
        !(isDebugPath && (rawPath == "logs")) {

Reply:
Done